### ISSUE TITLE
class SelfAttentionRelative(nn.Module):

### ISSUE THREAD

**Issue by EltonQm** [pbloem/former#40]
former/modules.py
line 469
 def __init__(self, emb, pos_embedding, heads=8, mask=False, ):

There should be no comma after the mask argument

**Comment by kkpo4553** [comment]
Hello, we tried to solve the issue.

This is what we did:

Removed the trailing comma after the 'mask' parameter in the __init__ method of the SelfAttentionRelative class to fix the described issue.

You can review changes in this commit: https://github.com/kkpo4553/pbloem-former-40/commit/522353a74402e2373df61ac66333d2c3f3461e21.


> [!CAUTION]
> Disclaimer: The concept of solution was created by AI and you should never copy paste this code before you check the correctness of generated code. Solution might not be complete, you should use this code as an inspiration only.

---

If you no longer want Latta AI to attempt solving issues on your repository, you can block this account.
